manticore/core/state.py

```diff
@@ -138,40 +134,6 @@ def __exit__(self, ty, value, traceback):
         self._child = None
         self.platform.constraints = self.constraints
 
-    def execute(self):
```

Comment:
Moved to `native/state.py:State`. The `ethereum/state.py:State` has its own implementation that just does `return self._platform.execute()` for now.